packages/app/cypress/e2e/top-nav.cy.ts

```diff
@@ -205,6 +207,35 @@ describe('App Top Nav Workflows', () => {
         cy.findAllByRole('dialog').should('not.exist')
       })
     })
+
+    context('version data unreachable', () => {
+      it('treats unreachable data as current version', () => {
+        cy.withCtx((ctx) => {
+          const oldFetch = ctx.util.fetch
+
+          // @ts-ignore sinon is a global in the node process where this is executed
+          sinon.stub(ctx.util, 'fetch').get(() => {
```

Comment:
would you be able to use a spy on fetch and verify it wasn't called with either url?

Reply:
@emilyrohrbough I actually am expecting these to be called with those urls and want to throw an error when they are.